front/src/store/index.js

```diff
@@ -13,6 +14,6 @@ export default new Vuex.Store({
   actions: {
   },
   modules: {
-    сelebrities
+    сelebrities, ReristrationUser
```

Comment:
wasn't it cinfusing for you that celebrities starts from small letter and your from upper?
please, be more careful with naming. we have standard